larray/core/array.py

```diff
@@ -2462,12 +2464,15 @@ def dump(self, header=True, wide=True, value_name='value', light=False, axes_nam
             Assuming header is True, whether or not to include axes names. If axes_names is 'except_last',
             all axes names will be included except the last. Defaults to True.
         na_repr : any scalar, optional
-            Replace missing values (NaN floats) by this value. Default to 'as_is' (do not do any replacement).
+            Replace missing values (NaN floats) by this value. Defaults to 'as_is' (do not do any replacement).
         maxlines : int, optional
             Maximum number of lines to show. Defaults to -1 (all lines are shown).
         edgeitems : int, optional
             If number of lines to display is greater than `maxlines`, only the first and last `edgeitems` lines are
             displayed. Only active if `maxlines` is not -1. Defaults to 5.
+        _axes_display_names : bool, optional
```

Comment:
are _parameters included in the api doc? If yes, I don't think we should document them the normal way, to avoid cluttering the api doc, but should rather add a comment with that info.